frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx

```diff
@@ -371,6 +374,7 @@ export function LineGraph_({
                 },
             },
             plugins: {
+                stacked100: { enable: showPercentStackView, precision: 1 },
```

Comment:
What does precision do here?

If we had a graph where the values were all below 1 would this break them?

(you can make a formula series divided by 10 or 100 maybe to test?)

Reply:
Hi @pauldambra, The precision is just the decimal number after the conversion into percentage. So if the values are 33.333% and 66.666% and if precision is 1, then the values will be 33.3% and 66.7%. If precision is 2, then the values will be 33.33% and 66.67%. So, no even if all the values are below 1, that does not have to do anything with precision value. We are safe :). In fact, the default value of precision is 1. I have read the docs.

Reply:
Awesome, thank you!